spec/spec_helper.rb

```diff
@@ -1,2 +1,8 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'trix'
+require 'rails/version'
+require 'action_dispatch'
+require 'action_controller/metal/live'
+require 'action_controller/test_case'
```

Comment:
Wouldn't just requiring `rspec/rails` make all of these other requires not needed?